src/pages/organize/[orgId]/people/[personId]/index.tsx

```diff
@@ -56,10 +56,10 @@ const PersonProfilePage: PageWithLayout<PersonProfilePageProps> = ({
       </Head>
       <Grid container direction="row" spacing={6}>
         <Grid item lg={4}>
-          <PersonDetailsCard {...{ person }} />
+          <PersonDetailsCard {...person} />
```

Comment:
My gut reaction to this is that it's a weird way to pass props. This basically means that you could use `PersonDetailsCard` like this, which I think is weird, because it would break the regular style of the codebase, and also ties the interface of the component to the interface of the API.

```tsx
<PersonDetailsCard first_name="Richard" last_name="Olsson" /> // etc
```

Why not just pass a regular person object, like this?

```
<PersonDetailsCard person={person}/>
```